dev-libs/svrcore/svrcore-4.1.2.ebuild

```diff
@@ -0,0 +1,34 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit multilib autotools
+
+DESCRIPTION="Mozilla LDAP C SDK"
+HOMEPAGE="http://www.port389.org/"
+SRC_URI="http://www.port389.org/binaries/"${P}".tar.bz2"
+
+LICENSE="MPL-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+DEPEND=">=dev-libs/nss-3.11
+	>=dev-libs/nspr-4.6"
+
+RDEPEND="${DEPEND}"
+
+PATCHES="${FILESDIR}/${PN}-4.1-gentoo.patch"
+
+src_prepare() {
+	default
+	eautoreconf
+}
+
+src_install() {
+	default
+	# cope with libraries being in /usr/lib/svrcore
+	echo "LDPATH='${EROOT}usr/$(get_libdir)/${PN}'" > "${T}/08svrcore" || die "Unable to create env file"
```

Comment:
I originally had '/' and Amynka suggested I change it to EROOT.  I also had it in pkg_preinst and was told this was bad form.  Can gentoo devs please come to a consensus on coding style before making me change ebuilds, undo changes, change again.  This back and forth of conflicting advice is a huge time waster and incredibly pedantic and annoying.  I appreciate what you are all trying to do but you are also all in contradiction to each other and it's driving us proxy-maint'd users insane.